docs/addons/konveyor.md

```diff
@@ -0,0 +1,213 @@
+# Konveyor Add-On for Amazon EKS Blueprints
+
+Streamline the modernization of your application portfolio to Kubernetes. The Konveyor tool gives you insight throughout the migration process - whether you're making decisions at the portfolio or application level.
```

Comment:
Same feedback as patterns repo. Start with What is Konveyor and what is its value proposition to customers? Check our other partner addons for example